README.md

```diff
@@ -50,3 +50,18 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Changes for module 2
+
+New functions written to manipulate items in the ToDo app which make API calls to Trello where the items are now stored. Functions are in \data\trello_items.py
+Corresponding updates made to \app.py and \templates\index.html 
+
+Trello API key and token, as well as other necessary secrets are stored in trello_config.py which has been included in .gitignore
```

Comment:
Thanks for the clear instructions here! This is fine here, but will become more fiddly as we progress, e.g. how will we inject these secrets onto deployed servers, or switch the secrets out when we're running tests? The typical solution is with environment variables, so I advise switching to use the `.env` file provided - you can put the same values in that file (and the keys without values as below in the `.env.template` file, and then access them in the code with e.g. `os.getenv('TRELLO_URL')`